react/Checkbox/Checkbox.js

```diff
@@ -22,10 +22,10 @@ export default class Checkbox extends Component {
 
   static propTypes = {
     id: PropTypes.string.isRequired,
-    label: PropTypes.string.isRequired,
+    label: PropTypes.node.isRequired,
     className: PropTypes.string,
     inputProps: PropTypes.shape({
-      onChange: PropTypes.func.isRequired,
```

Comment:
It'd be great to make this prop type a function that checks if the label is a string and makes the prop required if it is